src/model/submissions.model.ts

```diff
@@ -0,0 +1,111 @@
+import {
+    // JoinColumn, // TODO: Uncomment this with FK constraint once the course entity is merged
+    //ManyToOne, // TODO: Uncomment this with FK constraint once the course entity is merged
+    Entity,
+    Column,
+    PrimaryGeneratedColumn,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn
+} from "typeorm";
+
+
+//import UserModel from './users.model' // TODO: Uncomment this with FK constraint once the course entity is merged
+
+@Entity('submissions')
+export default class SubmissionModel {
+    @PrimaryGeneratedColumn()
+    id: number      //submission id, change to submission id?
+
+    @CreateDateColumn({name: "created_at"})
+    createdAt: Date
+
+    @UpdateDateColumn({name: "updated_at"})
+    updatedAt: Date
+
+    @DeleteDateColumn({name: "deleted_at"})
+    deletedAt?: Date
+
+    @Column({name: "submission_id"})
```

Comment:
I don't think you need a `submission_id` on the submission table. That's what the id is here.